src/02-crm/companies.apib

```diff
@@ -200,7 +200,7 @@ Update a company.
         + bic: `BICBANK` (string, optional, nullable)
         + language: `en` (string, optional, nullable)
         + responsible_user_id: `0ea94804-401d-4dbd-a577-c2d60998f798` (string, optional, nullable)
-        + remarks: `Met at expo` (string, optional, nullable)
+        + remarks: `Met at expo` (string, optional, nullable) - Uses markdown formatting
```

Comment:
B: Updating a company through API v2 does not parse Markdown syntax yet. It's noted in a code comment and on my TODO list (after I removed the AuthenticatedPerson from the flow).

Reply:
If this doesn't happen, we have a bigger problem then that the documentation is not correct.

If so, this needs to be fixed asap imo.